jupyterlab_git/git.py

```diff
@@ -345,14 +351,34 @@ def read_notebook(content):
             else:
                 return nbformat.reads(content, as_version=4)
 
+        def remove_cell_ids(nb):
```

Comment:
Let's mark this trick

```suggestion
        # TODO Fix this in nbdime
        def remove_cell_ids(nb):
```